### PR TITLE
added copy-text button and edited reverse function

### PR DESCRIPTION
**Pull request by HamzaYehia**

### CODE REVIEW

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
Looks amazing!

But, can you make it so the text the displays reversed is in a color following the color scheme, orange or something?

But, it may have to be changed after #3 gets merged (which I will do)

**Comment by HamzaYehia** [comment]
> Looks amazing!
> 
> But, can you make it so the text the displays reversed is in a color following the color scheme, orange or something?
> 
> But, it may have to be changed after #3 gets merged (which I will do)

**Comment by KendallDoesCoding** [comment]
Please make the edits in this branch/PR itself.

**Comment by HamzaYehia** [comment]
> Please make the edits in this branch/PR itself.
> 

Ok, I'll do it

**Comment by zabihhaqqani** [comment]
great addition @HamzaYehia!

**Comment by HamzaYehia** [comment]
> great addition @HamzaYehia!

Thanks!

**Comment by KendallDoesCoding** [comment]
Just merged #3, can you fix the merge conflicts?

Thanks!

**Comment by KendallDoesCoding** [comment]
@HamzaYehia Please resolve conflicts.. I just merged #6

**Comment by HamzaYehia** [comment]
> @HamzaYehia Please resolve conflicts.. I just merged #6

Hi,
I resolved the confilcts and pushed a new commit with copy-text button to the PR.

**Review by KendallDoesCoding — COMMENTED** [review]
I liked the previous interface, that when the button "reverse" is tapped, the reversed text goes on the next line and the copy button after that, this too is good but can you change it back to that interface, that you made in this PR?

**Comment by HamzaYehia** [comment]
> I liked the previous interface, that when the button "reverse" is tapped, the reversed text goes on the next line and the copy button after that, this too is good but can you change it back to that interface, that you made in this PR?

I've just changed it back, you can check again now.

**Comment by HamzaYehia** [comment]
> I liked the previous interface, that when the button "reverse" is tapped, the reversed text goes on the next line and the copy button after that, this too is good but can you change it back to that interface, that you made in this PR?

I've just changed it back, you can check again now.

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
Don't display "Reversed text appears here.." just comment it!

File: index.html [hunk 1]
```diff
@@ -18,7 +18,11 @@ <h3 class="sub-heading">
     </h3>
     <form>
       <input id="text" type="text" name="rev" />
-      <input id="reverse-btn" type="button" value="REVERSE" onClick="reverse(this.form)" />
+      <input class="buttons" type="button" value="REVERSE" onClick="reverse(this.form)" />
+
+      <p id="reversed-text" class="sub-heading"> Reversed text appears here..</p>
```
Comment (KendallDoesCoding) [inline index.html]:
```suggestion
      <!--<p id="reversed-text" class="sub-heading"> Reversed text appears here..</p>-->

```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -18,7 +18,12 @@ <h3 class="sub-heading">
     </h3>
     <form>
       <input id="text" type="text" name="rev" />
-      <input id="reverse-btn" type="button" value="REVERSE" onClick="reverse(this.form)" />
+      <input class="buttons" type="button" value="REVERSE" onClick="reverse(this.form)" />
+
+      <!--<p id="reversed-text" class="sub-heading"> Reversed text appears here..</p>-->
```
Comment (KendallDoesCoding) [inline index.html]:
```suggestion
      <p id="reversed-text" class="sub-heading"> <!--Reversed text appears here..--></p

```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -18,7 +18,13 @@ <h3 class="sub-heading">
     </h3>
     <form>
       <input id="text" type="text" name="rev" />
-      <input id="reverse-btn" type="button" value="REVERSE" onClick="reverse(this.form)" />
+      <input class="buttons" type="button" value="REVERSE" onClick="reverse(this.form)" />
+
+      <p id="reversed-text" class="sub-heading"> <!--Reversed text appears here..--></p
```
Comment (KendallDoesCoding) [inline index.html]:
```suggestion
      <p id="reversed-text" class="sub-heading"> <!--Reversed text appears here..--></p>

```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
oops

**Review by KendallDoesCoding — APPROVED** [review]
Perfect!

**Comment by KendallDoesCoding** [comment]
Thanks for your contribution @HamzaYehia

**Pull request merged**